packages/slice-machine/components/ScreenshotChangesModal/Views/index.tsx

```diff
@@ -58,9 +71,58 @@ function VariationScreenshot({
     onHandleFile: generateSliceCustomScreenshot,
   });
 
+  const sliceView = useMemo(
+    () =>
+      slice
+        ? [
+            {
+              sliceID: slice.model.id,
+              variationID: variationID,
+            },
+          ]
+        : null,
+    [slice.model.id, variationID]
+  );
+
+  if (!sliceView) return null;
+
+  const openSimulator = () =>
+    checkSimulatorSetup(true, () =>
+      window.open(
+        `/${slice?.href}/${slice?.model.name}/${variationID}/simulator`
+      )
+    );
+
   return (
     <>
       <Flex sx={{ p: 2, pt: 0, minHeight: "48px" }}>
+        <Button
+          variant="buttons.secondarySmall"
```

Comment:
You don't need to specify `buttons.` here because of theme-ui variant groups!
https://theme-ui.com/components/variants